product_docs/docs/biganimal/release/free_trial/detail/create_a_cluster/create_cluster_portal.mdx

```diff
@@ -6,7 +6,7 @@ indexCards: none
 
 ## Navigating to the Create Cluster page
 
-1. Navigate to the [BigAnimal portal](https://freetrial.biganimal.com/). Sign in with [your account](../create_an_account) if you need to.
+1. Navigate to the [BigAnimal portal](https://portal.biganimal.com)). Sign in with [your account](../create_an_account).
```

Comment:
This PR won't take effect _until_ BAH is live, correct?

Reply:
the current plan is to publish this on June 15